authentication/oidc.go

```diff
@@ -83,10 +87,24 @@ func NewOIDC(logger log.Logger, configs []TenantOIDCConfig) (http.Handler, map[s
 	return r, middlewares, warnings
 }
 
+func getCookieForTenant(tenant string) string {
+	return fmt.Sprintf("observatorium_%s", tenant)
+}
+
+type OIDCProvider struct {
+	logger           log.Logger
+	provider         *oidc.Provider
+	client           *http.Client
+	config           OIDCConfig
+	cookieName       string
+	callbackRedirect string
```

Comment:
callback and redirect are usually synonyms in OAuth terminology. We should rather call this either "callbackURL" or "redirectURL"

Reply:
Agreed. Renamed to `redirectURL`.